.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Make sure you can reproduce with the latest version available**
+`pip install milatools --upgrade`
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
```

Comment:
Click? I think reproducing the bug is mostly covered by "What command did you run?"

Reply:
Hmm ya sorry copied paste without thinking herre